addon/components/edit-form.js

```diff
@@ -0,0 +1,51 @@
+import { inject as service } from "@ember/service";
+import Component from "@glimmer/component";
+import { task } from "ember-concurrency-decorators";
+
+import handleModelErrors from "ember-emeis/decorators/handle-model-errors";
+
+export default class EditFormComponent extends Component {
+  @service intl;
+  @service notification;
+  @service router;
+
+  get parentRouteName() {
+    return this.router.currentRoute.parent.name;
+  }
+
+  get listViewRouteName() {
+    return (
+      this.args.listViewRouteName ||
+      `${this.parentRouteName}.index`.replace("ember-emeis.", "")
```

Comment:
This might cause a problem if the engine is mounted with an `as`, right? Did you check this?

Reply:
Done :ok: